dkan.install

```diff
@@ -230,3 +230,22 @@ function dkan_update_7009() {
   colorizer_clearcache();
   cache_clear_all();
 }
+
+/**
+ * Disable deprecated dkan_dataset_api and enable open_data_schema_map.
+ */
+function dkan_update_7010() {
+  if (module_exists('dkan_dataset_api')) {
+    module_disable(array('dkan_dataset_api'));
+
+    //Clear endpoint cache.
```

Comment:
It looks like the update hook creates the odsm filecache when the module is enabled… i don’t think we want to do that

If someone doesn’t have cron set up to clear the cache regularly they will never see their endpoints updated